src/core/runtime/runtime.cc

```diff
@@ -32,6 +32,13 @@ Logger log_legate("legate");
 
 // This is the unique string name for our library which can be used
 // from both C++ and Python to generate IDs
+
+using LegateVariantImpl = void (*)(TaskContext&);
+/*static */ std::vector<std::pair<int64_t, LegateVariantImpl> > Core::opIDs = *(new std::vector<std::pair<int64_t, LegateVariantImpl> >());
+/*static */ std::vector<std::pair<int64_t, LegateVariantImpl> > Core::gpuOpIDs = *(new std::vector<std::pair<int64_t, LegateVariantImpl> >());
+/*static */ std::unordered_map<long, LegateVariantImpl> Core::cpuDescriptors = *(new std::unordered_map<long, LegateVariantImpl>());
+/*static */ std::unordered_map<long, LegateVariantImpl> Core::gpuDescriptors = *(new std::unordered_map<long, LegateVariantImpl>());
+
```

Comment:
these arrays store function pointers to the registered Legate tasks. 
An openMP analogue will be added before merge